lib/views/staging-view.js

```diff
@@ -586,6 +606,35 @@ export default class StagingView extends React.Component {
     menu.popup(remote.getCurrentWindow());
   }
 
+  showActionsMenu(event) {
+    event.preventDefault();
+
+    const menu = new Menu();
+
+    const selectedItemCount = this.state.selection.getSelectedItems().size;
+    const pluralization = selectedItemCount > 1 ? 's' : '';
```

Comment:
if we ever i18n our package, all our hard coded pluralization is going to be fun to convert....

Reply:
Yuuuup 😕